eng/pipelines/linux.yml

```diff
@@ -114,16 +114,29 @@ stages:
+              - alpineQueues: \(Alpine.314.Amd64.Open\)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20210910135833-1848e19+\(Alpine.312.Amd64.Open\)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20211214164128-aca80d8
+              - alpineArm64Queues: \(Alpine.313.Arm64.Open\)ubuntu.1804.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-arm64v8-20210910135808-8a6f4f3+\(Alpine.314.Arm64.Open\)ubuntu.1804.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-arm64v8-20210910135810-8a6f4f3
+              - linuxArm64Queues: \(Ubuntu.1604.Arm64.Open\)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-bfcd90a-20200127194925
+
+          - ${{ else }}:
+            - ${{ if eq(parameters.fullMatrix, 'false') }}:
+              - linuxDefaultQueues: Centos.7.Amd64+RedHat.7.Amd64+Debian.9.Amd64+Ubuntu.1604.Amd64+Ubuntu.1804.Amd64+SLES.15.Amd64+\(Fedora.29.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-helix-a12566d-20191210224553
+              - alpineQueues: \(Alpine.314.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20210910135833-1848e19
+
+            - ${{ if eq(parameters.fullMatrix, 'true') }}:
+              - linuxDefaultQueues: Centos.7.Amd64+RedHat.7.Amd64+Debian.9.Amd64+Ubuntu.1604.Amd64+Ubuntu.1804.Amd64+SLES.12.Amd64+SLES.15.Amd64+\(Fedora.29.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-helix-a12566d-20191210224553+\(Fedora.30.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-30-helix-4f8cef7-20200121150022+\(Ubuntu.1910.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-19.10-helix-amd64-cfcfd50-20191030180623+\(Debian.10.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64-bfcd90a-20200121150006
+              - linuxArmQueues: \(Debian.9.Arm32\)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-bfcd90a-20200121150037
+              - alpineQueues: \(Alpine.314.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20210910135833-1848e19+\(Alpine.312.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20211214164128-aca80d8
+              - alpineArm64Queues: \(Alpine.313.Arm64\)ubuntu.1804.armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-arm64v8-20210910135808-8a6f4f3+\(Alpine.314.Arm64\)ubuntu.1804.armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-arm64v8-20210910135810-8a6f4f3
+              - linuxArm64Queues: \(Ubuntu.1604.Arm64\)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-bfcd90a-20200127194925
```

Comment:
I went through and checked .Open and not have parity, and that before and after have parity (aside from replacing Alpine 3.9 with Alpine 3.14).  No one else has to do that multi-diff if they don't want to.